tested it locally with java 17 on windows and got a build failure:
```
[ERROR] Failed to execute goal org.codehaus.mojo:ianal-maven-plugin:1.0-alpha-1:verify-legal-files (verify-legal-files)on project org.apache.sling.servlets.get:
Execution verify-legal-files of goal org.codehaus.mojo:ianal-maven-plugin:1.0-alpha-1:verify-legal-files failed:
Unable to make private java.io.File(java.lang.String,java.io.File) accessible: module java.base does not "opens java.io" to unnamed module @6a938034 -> [Help 1]
```
a rather strange error from this plugin.

got same problem when compiling with java 17 on linux.

--- a/pom.xml
+++ b/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <groupId>org.apache.sling</groupId>
         <artifactId>sling-bundle-parent</artifactId>
-        <version>41</version>
+        <version>48</version>
         <relativePath />
     </parent>
 
@@ -45,6 +45,7 @@
 
     <properties>
         <site.jira.version.id>12314473</site.jira.version.id>
+        <project.build.outputTimestamp>1</project.build.outputTimestamp>
     </properties>
 
     <build>

